webpack.config.js

```diff
@@ -1,43 +1,18 @@
-const commonConfig = {
-  resolve: {
-    extensions: [".ts", ".tsx", ".js"]
-  },
-  devtool: "source-map",
-  module: {
-    rules: [
-      {
-        test: /\.tsx?$/,
-        loader: "ts-loader"
-      }
-    ]
-  }
-};
-
-const outputPath = __dirname + "/vega/static";
-const outputLibraryTarget = "amd";
+const version = require('./package.json').version;
+const path = require('path');
 
 module.exports = [
-  // the main vega extension
-  Object.assign({}, commonConfig, {
+  {
     entry: "./src/index.ts",
     output: {
       filename: "index.js",
-      library: "nbextensions/jupyter-vega/index",
-      path: outputPath,
-      libraryTarget: outputLibraryTarget
-    }
-  }),
-  // the widget extension
-  Object.assign({}, commonConfig, {
-    entry: "./src/widget.ts",
-    output: {
-      filename: "widget.js",
-      path: outputPath,
-      libraryTarget: outputLibraryTarget
+      path: path.resolve(__dirname, "dist"),
```

Comment:
Why is this better than outputting right into /vega/static? The change here means that we cannot run webopack in watch mode.

Reply:
8e13aa7 should address this issue

Reply:
That's great! Can you explain why it's better to move the vega.js file out of the static directory, why we should remove the static files, and why we need to copy files into dist?

Reply:
> why it's better to move the vega.js file out of the static directory

I guess we may not need to copy it to `dist/` if it's only intended to be used with the python package.

> why we should remove the static files

I actually can't remember exactly why these have been removed, I'll look into adding them back

> why we need to copy files into dist

Mainly because it's a standard practice

Reply:
> Mainly because it's a standard practice and I'm slightly worried it may break some hard coded ipywidgets code (I've been learning the hard way that if you deviate at all from the ipywidgets cookiecutter template, it's likely to break things in surprising ways)

Ahh, I didn't know the standard. Thank you! This is really the only ipywidget I maintain.